### ISSUE TITLE
Feature: Average fuel consuption per 100 km

### ISSUE THREAD

**Issue by thehijacker** [AlfHou/hammond#94]
Just noticed that this is not calculated and it would be very useful information. To compare it with previous months for example. This are the data I get from current site I use to input my data:

![image](https://github.com/AlfHou/hammond/assets/13857043/c2a75b27-06fc-42b8-963b-aea28d8c9eab)

Are any of this data already avaialable or can it be added?

**Comment by DonRohan** [comment]
>     * show total average usage l/100 km (based on all gas records)
>     * show total average price of gas per 100 km (based on all gas records)
yes but a new entry that references distance driven would need to be implemented otherwise data would be false if one record is missing

>     * add option to input specific gas station when inputting gas record
would be great if these will be remembered in a drop-down kind of menu

>     * show total average usage l/100 (grouped by gas stations)
just out of curiosity: why grouped by gas station?

**Comment by Goloso98** [comment]
> just out of curiosity: why grouped by gas station?

maybe he wants to know the fuel quality.
